lib/widget/box.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:ensemble/framework/scope.dart';
+import 'package:ensemble/framework/widget/widget.dart';
+import 'package:ensemble/widget/helpers/controllers.dart';
+import 'package:ensemble/widget/helpers/widgets.dart';
+import 'package:ensemble_ts_interpreter/invokables/invokable.dart';
+import 'package:flutter/material.dart';
+
+import '../framework/view/page.dart';
+
+class Box extends StatefulWidget
```

Comment:
Is it intended to include box in this PR?

Reply:
Currently, we don't have any widgets to create empty boxes with styles like backgroundColor, borderRadius, etc. So I created one for it. So we can design our own shimmer layout using the Box widget instead of showing images for the loading state.

Reply:
So we've been refrained to add a single-use widgets like this. I modifed it to be more useful to draw multiple shapes here https://github.com/EnsembleUI/ensemble/pull/782